### PR TITLE
Add hyperclick support

### PR DESCRIPTION
**Pull request by cannoneyed**
Adds support for the fabulous `hyperclick` library (https://atom.io/packages/hyperclick) (https://github.com/facebooknuclideapm/hyperclick). Overrides the default path jump behavior to use jedi.

### CODE REVIEW

**Comment by MichaelAquilina** [comment]
Thanks for this! I will give this a look as soon as I get the chance.

Could you squash your commits to just a single one in the meanwhile?

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "python-tools",
   "main": "./lib/python-tools",
-  "version": "0.6.8",
+  "version": "0.7.0",
```
Comment (MichaelAquilina) [inline package.json]: could you revert this change please? I will find increase the version when I find the right number of changes to include with the next release :)

**Comment by MichaelAquilina** [comment]
@cannoneyed polite ping

**Comment by cannoneyed** [comment]
@MichaelAquilina Hey sorry I've been out for the last week - let me actually triple check that this is something that's worth merging - I've actually made a few changes to my python workflow and have been solely relying on [autocomplete-python](https://github.com/sadovnychyi/autocomplete-python) which already has built-in hyperclick support - I was finding that a number of types of path resolutions weren't being handled properly by Jedi in this PR, and this was actually crashing the plugin... Anyway, give me a few days to make sure that everything's square with this and I'll be happy to clean it up as you've proposed

**Pull request closed without merging**